service/raft/group0_state_machine.cc

```diff
@@ -141,20 +141,27 @@ static future<> write_mutations_to_database(storage_proxy& proxy, gms::inet_addr
 }
 
 group0_state_machine::modules_to_reload group0_state_machine::get_modules_to_reload(const std::vector<canonical_mutation>& mutations) {
+    using sys = db::system_keyspace;
     modules_to_reload modules;
 
     for (auto& mut: mutations) {
-        if (mut.column_family_id() == db::system_keyspace::service_levels_v2()->id()) {
+        auto id = mut.column_family_id();
+
+        if (id == sys::service_levels_v2()->id()) {
             modules.service_levels_cache = true;
```

Comment:
If the service levels table is modified then we need to update the cache, right? I think that `modules.service_levels_effective_cache` should be set to `true` in that case.